### PR TITLE
updates/jan-13-kajal

### PR DESCRIPTION
**Pull request by kkaajjaall**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -133,7 +134,9 @@ function () {
                 Route::post('/staffs/{id}/permissions', [StaffPermissionController::class, 'store']);
                 Route::get('/staffs/{id}/permissions/edit', [StaffPermissionController::class, 'edit']);
                 Route::put('/staffs/{id}/permissions', [StaffPermissionController::class, 'update']);
-
+                // search
+                Route::get('/search/courses', [CoursesController::class, 'search']);
+                Route::get('/search/courseMajors', [CoursesController::class, 'search']);
```
Comment (ithqindia) [inline routes/web.php:139]: courseMajors -> course-majors

File: app/Http/Controllers/CoursesController.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\User;
```
Comment (ithqindia) [inline app/Http/Controllers/CoursesController.php:5]: Remove this

**Review by ithqindia — CHANGES_REQUESTED** [review]

**Comment by kkaajjaall** [comment]
date picker is remain for ielts form

**Review by ithqindia — APPROVED** [review]

**Comment by ithqindia** [comment]
Merged locally in https://github.com/ithqindia/metroclub/pull/97

**Pull request closed without merging**